doc/sphinx/changes.rst

```diff
@@ -8,6 +8,477 @@ Recent changes
+  by Hugo Herbelin).
+- **Fixed:**
+  `Set Typeclasses Strict Resolution` is no longer ignored in
+  `typeclasses eauto with <dbs>`
+  (`#19436 <https://github.com/coq/coq/pull/19436>`_,
+  fixes `#15432 <https://github.com/coq/coq/issues/15432>`_,
+  by Jan-Oliver Kaiser).
+- **Fixed:**
+  Unbound variables were sometimes generated when a metavariable of a
+  theorem given to :tacn:`apply` occurred in the type of the theorem
+  under a :n:`fun`
+  (`#19769 <https://github.com/coq/coq/pull/19769>`_,
+  fixes `#17314 <https://github.com/coq/coq/issues/17314>`_,
+  by Hugo Herbelin).
+- **Fixed:**
+  `cbn` now considers primitive literals (integers, floats, arrays, strings)
+  "constructors", i.e. they now satisfy the `!` modifier in `Arguments`
+  (`#20004 <https://github.com/coq/coq/pull/20004>`_,
+  fixes `#20003 <https://github.com/coq/coq/issues/20003>`_,
+  by Jan-Oliver Kaiser).
+
+
+Ltac2 language
+^^^^^^^^^^^^^^
+- **Added:**
+  Added Ltac2 bindings for congruence and simpl congruence, it fixes #14289 not entirely but provides Ltac2 bindings for one of the tactics listed there
+  (`#19032 <https://github.com/coq/coq/pull/19032>`_,
+  fixes `#14289 <https://github.com/coq/coq/issues/14289>`_,
+  by Benny Smit, review of Jason Gross, review of Pierre-Marie Pédrot, review of Gaëtan Gilbert).
+- **Added:**
+  APIs `compare` `of_int` and `print` in `Ltac2.Uint63`
+  (`#19197 <https://github.com/coq/coq/pull/19197>`_,
+  by Gaëtan Gilbert).
+- **Added:**
+  :cmd:`Ltac2 Type` supports deprecation of the declared constructors
+  (`#19575 <https://github.com/coq/coq/pull/19575>`_,
+  by Gaëtan Gilbert).
+- **Added:**
+  `Ltac2.Constr.noccur_between` and `noccurn` to test for non-occurrence of local variables in terms
+  (`#19614 <https://github.com/coq/coq/pull/19614>`_,
+  by Gaëtan Gilbert).
+- **Deprecated:**
+  `Ltac2.Constr.occur_between` and `occurn` whose return values are the opposite of that implied by their names
+  (`#19614 <https://github.com/coq/coq/pull/19614>`_,
+  by Gaëtan Gilbert).
+- **Added:**
+  `Ltac2.Control.hyp_value` to get the value (`v` in `H := v`) of an hypothesis
+  (`#19630 <https://github.com/coq/coq/pull/19630>`_,
+  by Gaëtan Gilbert).
+- **Fixed:**
+  resolution of :ref:`abbreviations <Abbreviations>` in :n:`reference`
+  in :token:`ltac2_quotations`, for instance in eval tactic
+  delta-reduction flags :token:`ltac2_delta_reductions`
+  (`#19589 <https://github.com/coq/coq/pull/19589>`_,
+  fixes `#19590 <https://github.com/coq/coq/issues/19590>`_,
+  by Pierre Roux).
+- **Fixed:**
+  `Ltac2 Eval` does not require to be focussed in a goal
+  anymore (`#19961 <https://github.com/coq/coq/pull/19961>`_, by
+  Daniil Iaitskov).
+
+SSReflect
+^^^^^^^^^
+
+- **Changed:**
+  The :tacn:`done` tactic now tries to apply `sym_equal` with four arguments
+  instead of trying first with zero to three arguments.
+- **Changed:**
+  `done` uses `simple refine` instead of `apply` to apply `sym_equal`
```

Comment:
I guess
```suggestion
- **Changed:**
  `done` uses `simple refine` instead of `apply` to apply `sym_equal`
  (`#19399 <https://github.com/coq/coq/pull/19399>`_,
  by Quentin Vermande).
```
@Tragicus do you confirm?